### PR TITLE
Initial work on badge markdown generator. Ref #16

### PR DESCRIPTION
**Pull request by ellisonleao**
Initial thoughts here. Current implementantion is making the following changes:

- moved the nav badges to use shields.io using a different layout
- added a badge description text in main section with a modal link that generates a html and md templates for the badge for the current repository in the main input
- Caveat: current custom badge uses a base64 logo url, which is making the code verbose

Some screenshots:

### new badges
Screenshot from 2019-10-08 00-54-00

## modal after click
Screenshot from 2019-10-08 00-54-10

### CODE REVIEW

**Comment by rodrigo-brito** [comment]
Hi @ellisonleao, great job! It looks great!
Can you resolve the code conflict?

**Comment by ellisonleao** [comment]
done @rodrigo-brito

File: src/utils.js [hunk 1]
```diff
@@ -24,4 +24,8 @@ export function openGithubEvent(url) {
...
...mae3hhxVvZm6L3L3CIxR9wJTm5Mi7Zxwr35YWk1MlwLdf0S51nPPYgD893cns1tvHNx7Hspn5RzM7rvy/VuaDMSKsdrwz1MUMTfWMAsLbKYHhh+t7v7pFe6jWRHyZZdhNb3ng893eDN/8oSjD4y/5QzHo98Xb2zFKd8QPSUf6CSg/DNSVr+I/0axTp6XonFmsMUAT3D9CY61PTo5EMEYuAcPCX3Danny9HX13zBLH4pswonHhQjRD81xAvz44pTufl/RUpsjqorkhl+bgC3q3xUG8Ism7eYrFNvN7wRKYhMzffFt3ocM53rqKsfBtC8bpN8IEynSuGeinr7uTmkOPkWAb7wBBVwORynWcH6rxxyOTDdhsBvBZwWGvKXrzDiRJou4p7r3P67wCAi07ezYbPbsVf9JsklPar3DTMS02hlr0PkG1TlcPL6kxiKDB1iM7UIYmKYX8QY22r3fOcUPBWLq3Z3dcY2b9FmDJsKdHI012h9vIwL0tPLkw3PhcHyCxG5nJ/rjqxyNNMG7I0m3m5fYyxc+uVxM1GJIwr0lBEX2XsOHI8WwTklEIC4mYj722+MhfTcnPAtd+LcKh5Oqa5M//wPt70yPF+y9zJ4abp3DQ98sU5AGDGaW00/fU8wzE3972C4vhXMB+iBJQKazMH/noec8a25WpW/75HmlV37IM/bTk/Eoms+tJyOM8IiUQiq04+tv18rhh/rD9D9vsd1JQpU8LA9wOBwDy/379IURQtZww43hx3ezfgOFYoFPpFcXHxPfm4N99fd8ji4uIlLS0tk03T3P3VVYHke03T3N3S0jK5uLh4Sb6xdVw/b6msrNy6YsWK2o6Ojlsc2z6Wzwr2L0ISh23bxzo6Om5ZsWJFbWVl5dbjeeJx/75n7ty55rRp0+7b+/FHNURCCzVpt/QNYvljhO7YLfuPhhbu2bOnZtq0affNnTvX5H+VnFTLq+vXeVtbW+dEo9H1tm3HZS/52bao3NViyUxyzeao3H0k+bpt2/FoNLq+tbV1zl1/fMnL6An8n5HGxsay1tbWmeFweGk0Gt1yzaZQcFeL6WR0wKaos+tQPBiNRreEw+GlbW1tMxsbG8u+zDl+1b96UnZ++HGlV5GV1dXVZQUFBRpAMBg0Dxw4cOzhD82DD1w69hD0+xOEb+QbyVP+B2mHaLGyYdwnAAAAAElFTkSuQmCC';
```
Comment (rodrigo-brito) [inline src/utils.js]: It is tooooooo big. Maybe a small version of the logo in base64 can be a little smaller. It is too long for an URL icon.

**Comment by rodrigo-brito** [comment]
A suggestion, resize and otimize. This website can optmize this kind of image: http://b64.io/

**Comment by ellisonleao** [comment]
@rodrigo-brito moved to a 16x16px base64 image now. LMK if its okay

**Comment by rodrigo-brito** [comment]
It loogs great, Thank you @ellisonleao!
Can you rebase your PR to update it? A recent PR was merged in the front-end branch.

**Comment by ellisonleao** [comment]
hey @rodrigo-brito the current changes already contains that PR AFAIK. I've tried to rebase on my side, and i can't see any new changes. i've also updated the master branch on my fork as well

**Comment by rodrigo-brito** [comment]
You should follow the comands:
Add the original repository as remote:
- ` git remote add upstream git@github.com:rodrigo-brito/gocity.git`
- `git remote update upstream`
- `git rebase upstream/front-end`

After this, resolve the conflicts in file App.js, after.
- `git commit add .`
- `git commit rebase --continue`

I can make a cherry pick of your PR, but the Pull Request will be closed automatic by Github.

**Comment by ellisonleao** [comment]
Weird, i had already done those steps before and pushed the changes, with the branch feature. Anyway, just did it again now, LMK it its okay

**Comment by rodrigo-brito** [comment]
No problem, I will squash it.
Github will update in in merge. 
Vlw e flws! :call_me_hand:

**Comment by rodrigo-brito** [comment]
This conflict happens because a previous commit has conflict changes but the last don't. It blocks a fast foward `rebase and merge`, a squash solves well because it merge all commits, including the conflicting.

**Pull request merged**